frontend/public/reducers/features.ts

```diff
@@ -30,9 +30,13 @@ import { referenceForGroupVersionKind } from '../module/k8s';
 import { referenceForModel } from '../module/k8s/k8s-ref';
 import { RootState } from '../redux';
 import { ActionType as K8sActionType } from '@console/dynamic-plugin-sdk/src/app/k8s/actions/k8s';
+import { FeatureState } from '@console/dynamic-plugin-sdk/src/app/features';
 import { FeatureAction, ActionType } from '../actions/features';
 import { pluginStore } from '../plugins';
 
+// eslint-disable-next-line prettier/prettier
```

Comment:
why do you need to disable `prettier/prettier`  ?

Reply:
Prettier has some issues with type exports. This is due to a version thing. If we upgrade prettier to a newer version this should go away. I don't remember which version exactly though. I think you and I had hit this issue a while back.

Reply:
I don't see the problem in my dev environment. Can you provide more details or the issue you're seeing.

Reply:
https://github.com/prettier/prettier/issues/7263
The export type feature introduced in TS is supported starting prettier 2.0.